tests/platform_tests/test_chassis_reboot.py

```diff
@@ -0,0 +1,80 @@
+"""
+This test file is created for T2 chassis specific reboot test, need to skip for all T0/T1
+"""
+import pytest
+import random
+import logging
+import time
+from tests.common.reboot import wait_for_startup,\
+                                sync_reboot_history_queue_with_dut,\
+                                REBOOT_TYPE_HISTOYR_QUEUE
+from tests.platform_tests.test_reboot import check_interfaces_and_services
+
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,
+    pytest.mark.topology('t2')
+]
+
+
+def chassis_cold_reboot(dut, localhost):
+    logging.info(
+        "Sync reboot cause history queue with T2 reboot cause history queue")
+    sync_reboot_history_queue_with_dut(dut)
+
+    logging.info("Run cold reboot on {}".format(dut))
+    reboot_res, dut_datetime = dut.command("reboot")
```

Comment:
This returns an error: `ValueError: too many values to unpack (expected 2)`

Reply:
sorry I forgot to change here. I had the change in my local branch